client.go

```diff
@@ -225,11 +254,18 @@ func (client *Client) init() error {
 	if client.pixImage == nil {
 		return fmt.Errorf("PixImage is not set, use SetImage or SetImageFromBytes before Text or HOCRText")
 	}
+
 	C.SetPixImage(client.api, client.pixImage)
 
+	client.isInit = true
+
 	return nil
 }
 
+func (client *Client) flagForInit() {
```

Comment:
Unless it returns something represented by the method name, the method name should start with a verb.

Reply:
It looks like a verb to me, "To flag" 
Do you have a suggestion for a more appropriate name ?

Idea is to mark the instance (flag) to be init again next time it is required

Reply:
> It looks like a verb to me, "To flag"

Ah, you are right. With having the flag named `shouldInit`, the method name makes sense better for me. Thank you for pointing it out.